aten/src/ATen/Utils.cpp

```diff
@@ -12,4 +14,44 @@ int _crash_if_asan(int arg) {
   return x[0];
 }
 
+Tensor empty_cpu(IntArrayRef size, const TensorOptions& options_, c10::optional<c10::MemoryFormat> optional_memory_format) {
```

Comment:
Probably a good idea to have a little comment here explaining why this lives here and not in `native::`